luatest/justrun.lua

```diff
@@ -0,0 +1,164 @@
+local fun = require('fun')
+local json = require('json')
+local fiber = require('fiber')
+
+local log = require('luatest.log')
```

Comment:
I performed a little test:

```sh
./src/tarantool $(which luatest) test/app-luatest/override_panic_test.lua -v -c -l -
```

With `log`:

```lua
luatest | 2024-04-23 15:38:26.357 [19646] main/104/luatest/test.justrun I> Running a command: cd /tmp/XBCo7A && INPUTRC="/dev/null" /home/alex/projects/tarantool-meta/r/t-3/src/tarantool main.lua
```

With `luatest.log`:

```lua
luatest | 2024-04-23 15:37:39.270 [18624] main/104/luatest/luatest.log I> Running a command: "cd /tmp/1PUXlF && INPUTRC=\"/dev/null\" /home/alex/projects/tarantool-meta/r/t-3/src/tarantool main.lua"
```

As for me, it is nice to have `justrun` in the log message.

Reply:
Yes, this became obvious to me only after some time. Because we override the `log` function in the `luatest.log` module, we will always get messages like:

```lua
luatest | ... [...] main/104/luatest/luatest.log I> ...
```

Obviously, the `luatest.log` module needs improvements.